src/rise-twitter.js

```diff
@@ -18,7 +18,6 @@ export default class RiseTwitter extends HTMLElement {
   constructor() {
     super();
     this.isPreview = false;
-    this.id = this.id || this._generateComponentId();
     console.log('RiseTwitter', this.id);
```

Comment:
This can be moved to the `_handleConfigure` function or removed.

Reply:
@jungeunyoon I don't think we should generate ids on the component but if there is no id we should call done. Thoughts? cheers

Reply:
@rodrigopavezi Yes good point! Perhaps just validate the response in `_handleConfigure` to ensure that both it and screen name exists?

Reply:
@rodrigopavezi the `console.log` still hasn't been resolved, thanks.

Reply:
@jungeunyoon changed. cheers